robolectric/src/test/java/org/robolectric/shadows/ShadowActivityTest.java

```diff
@@ -118,23 +118,25 @@ public void shouldUseActivityLabelFromManifestAsTitleForActivityWithShortName()
 
   @Test
   public void createActivity_noDisplayFinished_shouldFinishActivity() {
-    ActivityController<Activity> controller = Robolectric.buildActivity(Activity.class);
-    controller.get().setTheme(android.R.style.Theme_NoDisplay);
-    controller.create();
-    controller.get().finish();
-    controller.start().visible().resume();
-
-    activity = controller.get();
-    assertThat(activity.isFinishing()).isTrue();
+      try(ActivityController<Activity> controller = Robolectric.buildActivity(Activity.class)) {
```

Comment:
I think you need to run google-java-format locally: https://github.com/robolectric/robolectric/wiki/Running-google-java-format.

Reply:
@utzcoz, can you please explain me the use of this google-java-format ?

Reply:
By the way, I have enabled the google-java-format plugin in the android studio, will that do ?

Reply:
You can follow above link to run google-java-format locally. We don't format all files of Robolectric and we only format changed lines/files.

Reply:
@utzcoz, what are these variables $HOME and $ROBOLECTRIC_ROOT, can I change that ?